draft-ietf-rats-architecture.md

```diff
@@ -734,10 +734,29 @@ or interval when changes in operational status, health, and so forth occur.
 
 ## Endorsements
 
-An Endorsement is a secure statement that some entity (e.g., a manufacturer) vouches for the integrity of the
-device's signing capability.  For example, if the signing capability is in hardware, then
-an Endorsement might be a manufacturer certificate that signs a public key whose corresponding
-private key is only known inside the device's hardware.  Thus, when Evidence and such an Endorsement
+An Endorsement input to a Verifier is always required. An Endorsement is what allows a Verifier to trust an Attester and in turn to trust the Evidence produced by an Attester. Without Endorsements a Verifier cannot function.
+Said another way, an Attester for which there is no Endorsement is not useful since there is no good basis by which a Verifier can trust it.
+
+One of the Endorsements for an Attester will just about always contain cryptographic key material.
+This key material will correspond to key material that is put in the Attester.
+The key material is the basis for the Verifier trusting the Attester.
+This architecture does not dictate any particular cryptographic protocol for establishing this trust, hence the key material can be symmetric or asymmetric, a single key, multiple keys, key hierarchies, X.509 cert or chains or other.
+
+Endorsements are produced by an Endorser.
+The Endorser is often the manufacturer of the device that contains the Attester, but not necessarily so.
+For example, it could be an OS vendor that supplies Attester key material to the device/Attester manufacturer.
+
+An Endorsement must be conveyed to the Verifier in a secure way so that the Verifier does not accept fake or false Endorsements that would result in trusting Attesters and Evidence that should not be trusted.
+This architecture does not dictate any particular means of securing this conveyance hence it might be a TLS connection, some sort of signed data, physically transferring the data or other.
```

Comment:
I don't see the point of calling this out for endorsements more than other messages.   (Personally I think it's already sufficiently covered in the doc for messages in general.)

Reply:
There are many sentences under Trust Model about the RP trusting the Verifier, more than I have here. 

There are also many sentences about the Endorser trusting the Verifier (the flip of what I am trying to convey) in the Trust Model section.

So there's precedent in the doc for text like I'm proposing and as far as I can see little text that is redundant with mine.

I'm OK taking this out if there's consensus. If we take it out maybe we consider taking out other text about other messages?